x/projects/module.go

```diff
@@ -141,6 +141,11 @@ func (am AppModule) RegisterServices(cfg module.Configurator) {
 	if err := cfg.RegisterMigration(types.ModuleName, 2, migrator.Migrate2to3); err != nil {
 		panic(fmt.Errorf("%s: failed to register migration to v3: %w", types.ModuleName, err))
 	}
+
+	// register v3 -> v4 migration
```

Comment:
No need if we combine into existing/upcoming v2 -> v3

Reply:
done